src/test/scala-2.12/org/apache/james/gatling/simulation/Configuration.scala

```diff
@@ -20,16 +20,12 @@ object Configuration {
   val WEBADMIN_PROTOCOL = Properties.envOrElse("WEBADMIN_PROTOCOL", "http")
   val BaseJamesWebAdministrationUrl = new URL(s"$WEBADMIN_PROTOCOL://$WebadminServerHostName:$WEBADMIN_PORT")
 
-  val DURATION_PROPERTY = Properties.envOrNone("DURATION") match {
-    case Some(duration) => Some(duration.toInt minutes)
-    case _ => None
-  }
-
-  val ScenarioDuration = DURATION_PROPERTY.getOrElse(1 hour)
-  val InjectionDuration = DURATION_PROPERTY.getOrElse(1 hour)
-  val UserCount = 100
-  val RandomlySentMails = 10
-  val NumberOfMailboxes = 10
-  val NumberOfMessages = 20
+  val InjectDuration = Properties.envOrElse("INJECT_DURATION", "10").toInt seconds
+  val ScenarioDuration = Properties.envOrElse("SCENARIO_DURATION", "10").toInt seconds
```

Comment:
That would potentially need to be synced with the bench before being merged I think? as we have a duration injected to simulate a warmup

Reply:
And likely the stress file test in james too